.github/workflows/R-CMD-check.yaml

```diff
@@ -16,38 +16,56 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: macos-latest,   r: 'release'}
-          - {os: macos-15,       r: 'next'}
-          - {os: windows-latest, r: 'release'}
-          - {os: windows-latest, r: 'devel'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
+          - { os: macos-latest,     r: 'release'  }
+          - { os: macos-15,         r: 'next'     }
+          - { os: windows-latest,   r: 'release'  }
+          - { os: windows-latest,   r: 'devel'    }
+          - { os: ubuntu-latest,    r: 'devel', http-user-agent: 'release' }
+          - { os: ubuntu-latest,    r: 'release'  }
+          - { os: ubuntu-latest,    r: 'oldrel-1' }
+          - { os: ubuntu-24.04-arm, r: 'release'  }
+          - { os: windows-11-arm,   r: 'release'  }
 
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
 
     steps:
       - name: Add more rust targets
-        if: runner.os == 'windows'
+        if: ${{ runner.os == 'windows' && runner.arch == 'X64' }}
         run: rustup target add x86_64-pc-windows-gnu
+      
+      - name: Install Rust on Arm Windows
+        if: ${{ runner.os == 'windows' && runner.arch == 'ARM64' }}
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > tmp.sh
+          sh ./tmp.sh -y --profile minimal --target aarch64-pc-windows-gnullvm
+          echo ${HOME}/.cargo/bin >> $GITHUB_PATH
+        shell: bash
 
       - uses: actions/checkout@v3
 
       - uses: r-lib/actions/setup-pandoc@v2
 
       - uses: r-lib/actions/setup-r@v2
+        # TODO: skip installation on Windows Arm64 and use the default installation
+        if: ${{ !(runner.os == 'windows' && runner.arch == 'ARM64') }}
```

Comment:
Doesn't this support windows-11-arm? 🤔
https://github.com/r-lib/actions/blob/bd49c52ffe281809afa6f0fecbf37483c5dd0b93/setup-r/src/installer.ts#L474-L477